javascript/browser-apis/measuring-performance/navigation-timing-api.md

```diff
@@ -42,14 +29,21 @@ The Navigation Timing API provides timing information about the page load proces
 The API records the time at key points in the page load process and stores each event as a propriety of the `window.performance.timing` object.
```

Comment:
```suggestion
The API records the time at key points in the page load process and stores each event as a property of the `window.performance.timing` object.
```

The revision question (Line 86) also needs the same change, but I can't add a comment on that part of the code.